///Answer 1 git version 2.34.1

///Answer 2 user.name=hrussell725 user.email=hr131422@ohio.edu

///Answer 3 It pops up with a manual for git add

///Answer 4 
///On branch master

///No commits yet

///Untracked files:
/// (use "git add <file>..." to include in what will be committed)
///    README.md
///    answers.md

///Answer 5 
///On branch master

///No commits yet

///Changes to be committed:
///  (use "git rm --cached <file>..." to unstage)
///       new file:   README.md

///Untracked files:
///  (use "git add <file>..." to include in what will be committed)
///        answers.md

/// Answer 6 
///On branch master

///No commits yet

///Changes to be committed:
///  (use "git rm --cached <file>..." to unstage)
///        new file:   README.md
///        new file:   answers.md

///Answer 7 
///On branch master
///nothing to commit, working tree clean

///Answer 8 
///commit 92c09df3422c6f9ea09b6c59ef118947a83e9d74 (HEAD -> master)
///Author: hrussell725 <hr131422@ohio.edu>
///Date:   Wed Jan 24 18:51:11 2024 -0500

///    Initial commit

///Answer 9 
///On branch main
///Your branch is up to date with 'origin/main'.

///nothing to commit, working tree clean

///Answer 10  No

///Answer 11  To https://github.com/hrussell725/git-lab.git
/// ! [rejected]        main -> main (fetch first)
///error: failed to push some refs to 'https://github.com/hrussell725/git-lab.git'
///hint: Updates were rejected because the remote contains work that you do
///hint: not have locally. This is usually caused by another repository pushing
///hint: to the same ref. You may want to first integrate the remote changes
///hint: (e.g., 'git pull ...') before pushing again.
///hint: See the 'Note about fast-forwards' in 'git push --help' for details.

///Answer 12 Yes

///Answer 13 .  ..  .git  README.md  answers.md  git-lab-2




